org.eclipse.jface.text/src/org/eclipse/jface/text/source/ISourceViewerExtension6.java

```diff
@@ -0,0 +1,8 @@
+package org.eclipse.jface.text.source;
+
+import org.eclipse.jface.text.ITextViewerLifecycle;
+
+public interface ISourceViewerExtension6 {
```

Comment:
This shouldn't be necessary nowadays, we can instead consider using default methods in existing APIs.

Reply:
Removed